transformations/word_noise/transformation.py

```diff
@@ -0,0 +1,82 @@
+from random import Random
+from typing import Callable, List, Tuple
+
+import spacy
+
+from interfaces.QuestionAnswerOperation import QuestionAnswerOperation
+from tasks.TaskTypes import TaskType
+
+
+class WordNoise(QuestionAnswerOperation):
+    tasks = [TaskType.QUESTION_ANSWERING]
+    languages = ["en"]
+
+    CONCAT_POSITION_APPEND = "append"
+    CONCAT_POSITION_PREPEND = "prepend"
+    CONCAT_POSITION_ANY = "any"
+
+    def __init__(
+        self,
+        num_words: int,
+        concat_position: str,
+        remove_stop_words: bool = False,
+        seed: int = 0,
+        max_outputs: int = 1,
+    ):
+        super().__init__(seed, max_outputs=max_outputs)
+        self.n = num_words
+        self.concat_pos = concat_position
+        self.concat_function = self.get_concat_function()
+        self.spacy_model = spacy.load("en_core_web_sm")
```

Comment:
`from initialize import spacy_nlp`
`self.spacy_model = spacy_nlp if spacy_nlp else spacy.load("en_core_web_sm")`

Reply:
I'll make this change.